### PR TITLE
Hotfix/domain form

### PR DESCRIPTION
**Pull request by gloriateodoro**
## Contexto
Ao configurar um domínio não é permitido usar caracteres especiais, acentos e nem letras maiúsculas, porém a plataforma não avisa sobre essa regra nos campos de configuração e isso resulta num gargalo no momento da certificação. 

## Checklist
- [x] Desabilitar botão quando houver caracteres especiais no domínio
- [x] Adicionar observação em texto alertando sobre o uso de caracteres especiais e letras maiúsculas
- [x] Atualiza regex para rejeitar caracteres especiais e letras maiúsculas no domínio, mantendo o uso de hífen e números. 

obs: as alterações do arquivo tsconfig.json são as que permitiram, no meu ambiente, a execução do script que gera certificados.

### CODE REVIEW

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **lpirola**
You can assign the PR to them by writing `/assign @lpirola` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fbonde).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/nossas/bonde/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bnd-operator** [comment]
@gloriateodoro: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | 6e3fd5d08f8ef914de292f77ae1d0ae40bac8da3 | [link](http://dashboard-jx.nossas.tech/namespaces/jx/pipelineruns/nossas-bonde-pr-1487-pr-dl5qz) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by lpirola — APPROVED** [review]

**Pull request merged**
